lwgenerate/gcp/gcp_test.go

```diff
@@ -362,6 +329,364 @@ func TestGenerationOrganizationLevelConfigurationCustomIntegrationName(t *testin
 	assert.Equal(t, reqProvider(moduleImportOrganizationLevelConfigurationCustomIntegrationName), hcl)
 }
 
+func TestGenerationProjectLevelAuditLogCustomBucketName(t *testing.T) {
```

Comment:
Is it possible to table test these instead of individual tests?

Reply:
I like that idea.  My next task will be to add support for `terraform-gcp-gke-audit-log`, which will require more tests added to this file.  You ok with leaving this improvement for that PR?